deploy/charts/rook-ceph/templates/configmap.yaml

```diff
@@ -82,9 +88,23 @@ data:
   ROOK_CSIADDONS_IMAGE: {{ .Values.csi.csiAddons.image | quote }}
 {{- end }}
 {{- end }}
+{{- if .Values.csi.nfs }}
+  ROOK_CSI_ENABLE_CEPNHFS: {{ .Values.csi.nfs.enabled | quote }}
+{{- if .Values.csi.nfs.image }}
+  ROOK_CSI_NFS_IMAGE: {{ .Values.csi.nfs.image | quote }}
+{{- end }}
+{{- if .Values.csi.nfs.probe }}
+{{- if .Values.csi.nfs.probe.image }}
+  ROOK_CSI_NFS_PROBE_IMAGE: {{ .Values.csi.nfs.probe.image | quote }}
+{{- end }}
+{{- end }}
+{{- end }}
 {{- if .Values.csi.cephfsPodLabels }}
   ROOK_CSI_CEPHFS_POD_LABELS: {{ .Values.csi.cephfsPodLabels | quote }}
 {{- end }}
+{{- if .Values.csi.cephnfsPodLabels }}
```

Comment:
am more inclined to call this as NFS instead of cephnfs to keep it the same in all places.

Reply:
> am more inclined to call this as NFS instead of cephnfs to keep it the same in all places.

I think so too, replacing `cephNFS` with just `nfs` or `NFS` sounds okay to me. 
If anybody still wants the ceph prefix with nfs, please comment here 
or else i'll change in a day or two 
and follow the same for provisioner commits

@nixpanic @travisn @BlaineEXE

Reply:
as these changes are already under `.../rook-ceph/...` directories, use `nfs` instead of `cephnfs`, no need to repeat `ceph` everywhere

Reply:
changed to nfs everywhere